common-data/src/main/scala/org/genivi/sota/data/Interval.scala

```diff
@@ -0,0 +1,22 @@
+package org.genivi.sota.data
+
+import java.time.Instant
+
+case class Interval(start: Instant, end: Instant) {
```

Comment:
I mean, not using that field other than writing it to DB / reading it from DB.
The `def * = ...` of `UpdateRequestTable` does contain:
```
x.periodOfValidity.start, x.periodOfValidity.end, 
```
So `Interval` should be kept, although it doesn't see much actual use.